README.md

```diff
@@ -120,7 +120,7 @@ Data which is stored upon startup includes:
 
 * It is easiest to use the development virtual machine to work on this project. Follow [these instructions](RemoteDesktopToVM.md) to set up and connect to the virtual machine using your Gordon account.
 
-* If this is your first time on the virtual machine, you will need to clone this repository. You can do this by using Git Bash. It is possible that you will need to [add the SSH key to your Git account](https://help.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh). You will then need to `cd ssh` then `cat id_rsa.pub`, copy the output and go to your github settings and paste it in your SSH keys.
+* If this is your first time on the virtual machine, you will need to clone this repository. You can do this by using Git Bash. It is possible that you will need to [add the SSH key to your Git account](https://help.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh, it will guide you to `cd .ssh` then `cat id_rsa.pub`, copy the output and go to your github settings and paste it in your SSH keys.
```

Comment:
I think the ")" that was deleted is important for markdown to get the link right.